vehicle/mkvii/software/dashboard/dashboard_config.h

```diff
@@ -0,0 +1,38 @@
+#include "libs/adc/api.h"
```

Comment:
This import doesn't seem like it's being used. If that's the case, can you remove it from this file, your C file and the BUILD file? (same applies for the rest of your imports, not all are being used)